src/modules/ConsumerGroups/ConsumerGroups.tsx

```diff
@@ -79,6 +81,15 @@ const ConsumerGroups: React.FunctionComponent<ConsumerGroupsProps> = ({
     fetchConsumerGroups();
   }, [search, order]);
 
+  useEffect(() => {
+    //update setConsumerGroupDetail state after reset offset value
+    //so that values will update on page
+    const filteredGroup =
+      consumerGroups &&
+      consumerGroups.items?.filter((g) => g.groupId === groupId)[0];
```

Comment:
Adjusting array without checking size. 
Even if state would always have single item in array 
it is good to check size